src/modules/locale/locale.conf

```diff
@@ -1,7 +1,28 @@
 ---
+# This settings are used to set your default system time zone.
+# Time zones are usually located under /usr/share/zoneinfo and
+# provided by the 'tzdata' package of your Distribution.
+#
+# Distributions using systemd can list available
+# time zones by using the timedatectl command.
+#    timedatectl list-timezones
 region:                     "America"
 zone:                       "New_York"
 
-# GeoIP settings. Leave commented out to disable GeoIP.
-#localeGenPath:             "/etc/locale.gen"
+
+# System locales are detected in the following order:
+#
+# /usr/share/i18n/SUPPORTED
+#  localeGenPath ( when set )
+#   /etc/locale.gen
```

Comment:
This is not correct. The hardcoded `/etc/locale.gen` is not checked if `localeGenPath` is set.

The correct list is:
```
/usr/share/i18n/SUPPORTED
localeGenPath (defaults to /etc/locale.gen if not set)
'locale -a' output
```

Reply:
I what I meant .. Fixed now